### DISCUSSION TITLE
'Not authenticated with Minecraft.net' 'Initialization Failed'

### DISCUSSION THREAD

**Discussion by Suprxninja** [ORelio/Minecraft-Console-Client#2609] [Q&A]
I've used CC for years never had any issues I couldn't solve. Now, I've been dealing with this problem for months with no solution. Screenshot included.

Started out, when using CC on my laptop one day, it randomly stopped connecting. I quickly found out, for some reason, it would only connect using one of my two wifi networks on the same router (the 2G and 5G networks). If I was connected to the 5G network, it would give the error and not connect. It would only connect to the other network. So then I tested CC with another PC I use as a server, where I normally leave CC running all the time. On that PC it would still connect to any network, no problem... until it didn't. Suddenly after a couple weeks, that PC too gave the same problem, only connects on a particular network. I did nothing to that PC, no mods or changes in firewalls, settings, or anything that I didn't have before when it was working. It's literally sat there untouched before it stopped working.

I thought, no big deal, I'll just use the other network for now... But after a few weeks suddenly it too wouldn't connect to any network on that PC, period... So temporarily, I used CC again on my laptop, it connected fine, using that one network it would normally work on when this all started happening... and then again after a week or so, it wouldn't connect to any network. Now it won't connect using any of the two PC's.

My laptop was due for a factory reset, so I decided screw the diagnostics and wasting time, reset the laptop to factory, installed MC again and CC... Didn't change. Same problem. Won't connect to any network. I've tried different accounts. I've double checked the username/password, I haven't messed with the ini since it was working, besides account information, but still I've completely deleted everything and started over, but nothing... I even factory reset my router, since CC was picky about the networks, thought that might help, but nope, it changed nothing.

Both my accounts work just fine on the Microsoft Client on any network, any computer.

CC is connecting to the internet and my server, but having trouble authenticating with Microsoft. I have NO CLUE how it knew what wifi network I was on, that it would only connect to a particular one, considering it's still the same mac and ip's regardless... No one I know has ever had this issue. It's been going on for months now. CC has become unusable for me and there really isn't a second option. Anyone else run into this issue?

![cclient3](https://github.com/MCCTeam/Minecraft-Console-Client/assets/148988404/927fab1a-a9e2-4eb1-bd42-ccf2b0ca9188)

**Comment by milutinke** [comment]
Use the browser login method, if that does not work, use a proxy, Microsoft often blocks IP addresses/requests for some time for various reasons.
Also, it might be that the login via the console fails if you have enabled 2FA, maybe some time it requires 2FA and other it does not.
I'll look into this after I do the 1.20.2 update.

**Comment by breadbyte** [comment]
Can you try deleting the session file located in the Minecraft Client executable and try logging in again?
